orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js

```diff
@@ -204,7 +204,7 @@ module('Integration | Component | routes/authenticated/campaign/assessment/list'
   });
 
   module('when the campaign has badges', function() {
-    test('it should display badge column', async function(assert) {
+    test('it should display badge column when participant shared result', async function(assert) {
```

Comment:
**suggestion (non-blocking) :**
J'aurais rajouté les tests : 
- `it should display badge column when participant shared result` (orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js:207) 
- `it should display badge and tooltip when pariticipant shared his results` (orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js:236)

Dans le contexte `when a participant has shared his results` (orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js:15)


Et le test :
- `it should not display badge neither tooltip when participant not shared result` (orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js:266)

dans le contexte  `when a participant has not shared his results yet`  (orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js:52)

Reply:
done .